pkl/extensions/pkl.bzl

```
@@ -49,11 +49,11 @@ pkl_project = tag_class(
 )
 
 def _toolchain_extension(module_ctx):
-    pkl_cli_binaries()
+    cli_binaries = pkl_cli_binaries()
 
     workspaces = []
     seen_packages = []
-    direct_deps = []
+    direct_deps = [bin for bin in cli_binaries]
```

Is this list comprehension necessary?
```suggestion
    direct_deps = cli_binaries
```